smart_contract/executor/src/default/permissions.rs

```diff
@@ -85,63 +90,63 @@ macro_rules! declare_tokens {
 macro_rules! token {
```

Comment:
Consider also renaming `token!` and `declare_tokens!` macros in this file

Reply:
Done